pkg/build/api/v1/types.go

```diff
@@ -477,6 +477,13 @@ type SourceBuildStrategy struct {
 
 	// forcePull describes if the builder should pull the images from registry prior to building.
 	ForcePull bool `json:"forcePull,omitempty"`
+
+	// RuntimeImage is the optinal image that is used to run an application
+	// without unneeded dependencies installed.
+	RuntimeImage *kapi.ObjectReference `json:"runtimeImage,omitempty"`
+
+	// RuntimeArtifacts holds the mapping between the artifacts paths on the builder image and the paths on the runtime image where they should be copied.
+	RuntimeArtifacts []ImageSourcePath `json:"runtimeArtifacts,omitempty"`
```

Comment:
unless you mean add it to the config object that's getting passed from openshift, then yes.  but it won't be exposed as a user option.